src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -225,6 +230,9 @@ public static void deleteColony(final int id, final boolean canDestroy)
         {
             Log.getLogger().warn("Deleting Colony " + id + " errored:", e);
         }
+        @NotNull final File saveDir = new File(DimensionManager.getWorld(0).getSaveHandler().getWorldDirectory(), FILENAME_MINECOLONIES_PATH);
+        @NotNull final File file = new File(saveDir, String.format(FILENAME_COLONY, id));
+        file.delete();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Do something with the "boolean" value returned by "delete". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS899)